modules/sdk-coin-avaxp/package.json

```diff
@@ -51,6 +51,7 @@
     "@bitgo/sdk-core": "^26.3.0",
     "@bitgo/statics": "^48.1.0",
     "@bitgo/utxo-lib": "^9.34.0",
+    "bitgo-aaron-avalanchejs": "^4.0.5",
```

Comment:
If we know this is a working fix, can we fork this repo into a bitgo controlled repo?

Reply:
yes on it